service/prometheus/prometheustest/prometheustest.go

```diff
@@ -0,0 +1,12 @@
+// prometheustest provides a test implementation of the PrometheusReloader interface.
```

Comment:
Package doc should be in the form of `Package <package name> ... .`.

Reply:
Fixed.